colors/vim-monokai-tasty.vim

```diff
@@ -2,6 +2,7 @@
 "
 " Author: Patrick Stockwell <mail@deciphr.net>
 
+hi clear
```

Comment:
This seems pretty standard across vim colorschemes. I think it helps to avoid unexpected highlights, especially for groups not targeted by this colorscheme.